runtime/j9vm/jvm.c

```diff
@@ -3068,9 +3065,6 @@ restoreLibpath(J9LibpathBackup *libpathBackup)
 		libpathBackup->j9prefixLen = 0;
 	}
 }
-#endif /* AIXPPC */
-
-#if defined(AIXPPC)
```

Comment:
Please leave the blank line between functions.

Reply:
Fixed.